kivymd/uix/button/button.py

```diff
@@ -162,9 +163,9 @@ def on_start(self):
                     MDFloatingActionButton(
                         icon="pencil",
                         type=type_button,
-                        theme_text_color="Custom",
```

Comment:
This is the API change. Therefore, we have to specify the corresponding property (`deprecated`) - [deprecated=True](https://github.com/kivymd/KivyMD/blob/master/kivymd/uix/textfield/textfield.py#L474)

We can't just take and delete a property that users are using. We must warn users that this property will be removed in the future.

Reply:
The new M3 floating action buttons can take a text label, which means almost all buttons can take a text label/have a text label variant (only the Icon buttons don't).

Therefore this property, which is currently on BaseButton and therefore present even for those subclasses that don't use it like MDIcon (or currently MDFloatingActionButton), would remain rather than being deprecated? (it just wouldn't do anything if your button didn't have text).

In terms of backwards compatibility, everything still works as before - you can use text_color and theme_text_color to set the theme and text color of an MDIcon even as you did before, although it is more consistent to use icon_color and theme_icon_color as is used in the combined text and icon buttons.

Therefore I have marked this as deprecated on the MDIcon class only?